docs/Using-Pantheon/Transactions/Revert-Reason.md

```diff
@@ -0,0 +1,75 @@
+description: Including revert reason in transactions 
+<!--- END of page meta data -->
+
+# Revert Reason 
+
+In contracts, the [`revert`](https://solidity.readthedocs.io/en/v0.5.10/control-structures.html#revert) function 
+triggers an exception to flag an error and revert the current call. An optional string message containing 
+information about the error is passed back to the caller.
+
+!!! example
+    ```sol
+    pragma solidity >=0.5.0 <0.7.0;
+    
+    contract VendingMachine {
+        function buy(uint amount) public payable {
+            if (amount > msg.value / 2 ether)
+                revert("Not enough Ether provided");
+            // Alternative way to do it:
+            require(
+                amount <= msg.value / 2 ether,
+                "Not enough Ether provided."
+            );
+            // Perform the purchase.
+        }
+    }
+    ```
+
+## Enabling Revert Reason 
+
+Use the [`--revert-reason-enabled`](../../Reference/Pantheon-CLI-Syntax.md#revert-reason-enabled) command line option
+to include the revert reason in the transaction receipt in Pantheon. 
+
+!!! caution 
+    Enabling revert reason may use a significant amount of memory. We do not recommend enabling revert
+    reason when connected to the Ethereum mainnet. 
```

Comment:
same text as elsewhere? (I put a comment against it)

Reply:
Fixed